Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -310,6 +318,18 @@ open class ChatMessageListVC:
 
         cell.messageContentView?.delegate = self
         cell.messageContentView?.content = message
+        cell.dateSeparatorView.isHidden = true
+
+        let previousIndexPath = IndexPath(row: indexPath.row + 1, section: indexPath.section)
+        if let previousMessage = dataSource?.chatMessageListVC(self, messageAt: previousIndexPath),
```

Comment:
If there is no previous message, you should be showing the dataSeparator. Shouldn't you?

Reply:
I thought I shouldn't, but actually it makes sense to do it. Especially because our overlay disappears. Nice catch! Will change this 👍

Reply:
✅